docker/hwaccel.transcoding.yml

```diff
@@ -38,12 +38,6 @@ services:
       - /dev/dri:/dev/dri
       - /dev/dma_heap:/dev/dma_heap
       - /dev/mpp_service:/dev/mpp_service
-    volumes:
```

Comment:
Is this considered a breaking change? What happens if the user doesn't change this?

Reply:
it's not a breaking change.

if the user keeps an old version of this file that includes the mounts, these files are still mounted into the container but unused. no issue.